### DISCUSSION TITLE
Configure nginx with SSI and enable by default

### DISCUSSION THREAD

**Discussion by jrrdnx** [beyondcode/herd-community#629] [Ideas]
Looking to use server side includes and came across this pull request for Valet from 2018: https://github.com/laravel/valet/pull/517

Attempted to enable this for Herd but this results in an `unknown directive "ssi"` error in the logs. After some digging, it appears nginx is configured without `http_ssi_module` by default.

```
% /Applications/Herd.app/Contents/Resources/nginx-arm64 -V
nginx version: nginx/1.25.4
built with OpenSSL 3.2.1 30 Jan 2024
TLS SNI support enabled
configure arguments: [...] --without-http_ssi_module [...]
```

Would love to have this included and enabled by default.

**Comment by boynet** [comment]
did you find a solution for using ssi with herd?

**Comment by jrrdnx** [comment]
@boynet Not until they remove the `--without-http_ssi_module` argument.
